src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__staff_renewal_feed.sql

```diff
@@ -0,0 +1,98 @@
+with
+    step_scales as (
+        select
+            sr.employee_number,
+            pss.salary_rule,
+            pss.scale_cy_salary,
+            pss.scale_ny_salary,
+            pss.scale_step,
+            pss.school_level,
+        from {{ ref("base_people__staff_roster") }} as sr
+        left join
+            {{ source("people", "src_people__salary_scale") }} as pss
+            on sr.job_title = pss.job_title
+            and sr.business_unit_assigned_name = pss.region
+            and sr.base_remuneration_annual_rate_amount_amount_value + 150
+            >= pss.scale_cy_salary
+            and sr.base_remuneration_annual_rate_amount_amount_value + 150
+            < pss.scale_ny_salary
+            and pss.school_level is null
+        where sr.assignment_status = 'Active' and pss.school_level is null
+
+        union all
+
+        select
+            sr.employee_number,
+            pss.salary_rule,
+            pss.scale_cy_salary,
+            pss.scale_ny_salary,
+            pss.scale_step,
+            pss.school_level,
+        from {{ ref("base_people__staff_roster") }} as sr
+        left join
+            {{ source("people", "src_people__salary_scale") }} as pss
+            on sr.job_title = pss.job_title
+            and sr.business_unit_assigned_name = pss.region
+            and sr.base_remuneration_annual_rate_amount_amount_value + 150
+            >= pss.scale_cy_salary
+            and sr.base_remuneration_annual_rate_amount_amount_value + 150
+            < pss.scale_ny_salary
+            and pss.school_level = sr.home_work_location_grade_band
+        where
+            sr.assignment_status not in ('Terminated', 'Deceased')
+            and pss.school_level is not null

```

Comment:
you can move `pss.school_level is not null` to the join and make it an inner join

Reply:
Same as above: src_people__salary_scale is not an exhaustive list of jobs, so I don't want to inner join it